Dockerfile.multi-arch

```diff
@@ -0,0 +1,8 @@
+ARG OS="linux"
+ARG ARCH="amd64"
+FROM quay.io/prometheus/busybox-${OS}-${ARCH}:latest
```

Comment:
I don't think using `latest` here is a good idea. Maybe we should pin this to a SHA as we are doing in the primary `Dockerfile`

Reply:
Done.